agent/jvm/src/test/java/org/jolokia/jvmagent/security/KeyStoreUtilTest.java

```diff
@@ -59,6 +59,33 @@ public void testTrustStore() throws KeyStoreException, CertificateException, NoS
         assertEquals(key.getAlgorithm(),"RSA");
     }
 
+    @Test
+    public void testTrustStoreWithMultipeEntries() throws KeyStoreException, CertificateException, NoSuchAlgorithmException, IOException {
+        File caPem = getTempFile("ca/cert-multi.pem");
+        KeyStore keystore = createKeyStore();
+
+        KeyStoreUtil.updateWithCaPem(keystore, caPem);
+
+        Enumeration<String> aliases = keystore.aliases();
+        String alias = aliases.nextElement();
+        assertTrue(aliases.hasMoreElements());
+        assertTrue(alias.contains("another.test.jolokia.org"));
```

Comment:
It's an incorrect assumption of the test to require an `Enumeration` to be deterministically ordered. The contract of Enumeration does not mandate that (https://docs.oracle.com/javase/7/docs/api/java/util/Enumeration.html).

Reply:
Agreed. Instead it should be iterated over all elements and check whether  all elements are included (in any order).

Reply:
Sorry, I'll refactor the test. I was a little hash doing the PR.

Reply:
@rhusar thanks for the heads up ! I fixed the tests on master, should be fine now (no checkValidity(), don't imply order in enumerations)

Reply:
@dsimansk I just pushed an update. That might already addresses that issue. Could you please have a brief look ?

Reply:
@rhuss  @dsimansk  Looks good now!